demo/Makefile

```diff
@@ -31,7 +31,7 @@ install-cli:
 # Bootstrap the cluster
 # This should only be called once per cluster
 # Runs bootstrap in VM and cluster add on host
-bootstrap:
+bootstrap: install-cli
```

Comment:
We should update `install-cli` so it only installs the CLI if it doesn't exist (probably easiest to rename `install-cli` to `$(CLI_INSTALL_BIN)` so the `make` will do that).